python/tvm/driver/tvmc/runner.py

```diff
@@ -230,8 +235,8 @@ def make_inputs_dict(inputs_file, shape_dict, dtype_dict, fill_mode):
 
     Parameters
     ----------
-    inputs_file : str
-        Path to a .npz file containing the inputs.
+    inputs : dict
```

Comment:
```suggestion
    inputs : dict, or None
```
I'm actually not quite sure how this type should be in numpy-style...in Python style I'll just write `Optional[Dict]`, but an argument with this type is usually `inputs=None`. Meanwhile, a function called "make_inputs_dict" but having `inputs=None` is also weird...

Another direction is keeping `inputs` to be `dict`, and it becomes user's resposibility to pass `{}` when no inputs. In this case, you don't need the None checker.

Reply:
Hi Cody,
Thanks for the feedback. Just pushed an updated version as you suggested.